.github/workflows/release-code-freeze.yml

```diff
@@ -1,7 +1,7 @@
 name: 'Release: Code freeze'
 on:
     schedule:
-        - cron: '0 16 * * 4' # Run at 1600 UTC on Thursdays.
+        - cron: '0 23 * * 1' # Run at 2300 UTC on Mondays.
```

Comment:
Note: I also updated the time to 2300 UTC, as we only need to be able to cut the testing package sometime on Tuesday... this allows as much time as possible.